packages/components/socialFeed/NewsFeed/NewsFeedInput.tsx

```diff
@@ -306,15 +309,55 @@ export const NewsFeedInput = React.forwardRef<
             ],
           });
         } else {
-          await mutateAsync({
-            client,
-            msg,
-            args: {
-              fee: defaultSocialFeedFee,
-              memo: "",
-              funds: [coin(postFee, "utori")],
-            },
-          });
+          if (selectedNetwork?.kind === NetworkKind.Gno) {
+            // const provider = new GnoJSONRPCProvider(selectedNetwork.endpoint);
+
+            const msg = {
+              category: postCategory,
+              identifier,
+              metadata: JSON.stringify(metadata),
+              parentPostIdentifier: hasUsername ? replyTo?.parentId : parentId,
+            };
+
+            const vmCall = {
+              caller: selectedWallet?.address || "",
+              send: "",
+              pkg_path: GNO_SOCIAL_FEEDS_PKG_PATH,
+              func: "CreatePost",
+              args: [
+                TERITORI_FEED_ID,
+                msg.parentPostIdentifier || "0",
+                msg.category.toString(),
+                msg.metadata,
+              ],
+            };
+
+            const txHash = await adenaDoContract(
+              selectedNetworkId,
+              [{ type: "/vm.m_call", value: vmCall }],
+              {
+                gasWanted: 1_000_000,
```

Comment:
is this OK, hardcoded?

Reply:
yes, for this value, default is 1_000_000 so I can remove that